manim/container.py

```diff
@@ -0,0 +1,30 @@
+"""
+Abstract base class for several objects used by manim.  In particular, both
+:class:`~.Scene` and :class:`~.Mobject` inherit from Container.
+"""
+
+from abc import ABC, abstractmethod
+from .utils.config_ops import digest_config
+
+__all__ = ["Container"]
+
+
+class Container(ABC):
+    """Abstract base class for several manim objects.
+
+    Parameters
+    ----------
+    kwargs : :class:`dict`
```

Comment:
type of `kwargs` should be the type of the expected values.
So prob looking for `kwargs : Any` here (given the explanation in the description)